### ISSUE TITLE
Tenchu: return from darkness - Loading screen

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/game-compatibility#87]
_From @ergo720 on July 25, 2017 15:8_

The game shows a loading screen but crashes immediately after. Also, you can hear a sound after the loading screen but just before the crash.
PAL, win10, 3ca1a354

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/1173810/KrnlDebug.txt)
[Xbe.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/1185619/Xbe.txt)
![tenchu](https://user-images.githubusercontent.com/30000751/28579060-04de572a-714b-11e7-81bd-31433ce84e04.PNG)
![tenchu2](https://user-images.githubusercontent.com/30000751/28579063-0869b4b6-714b-11e7-99ed-b02095cf0981.PNG)

_Copied from original issue: Cxbx-Reloaded/Cxbx-Reloaded#518_

**Comment by ergo720** [comment]
Great news: with https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/commit/c609654f35cabf37d1e41ec514f5fd425e1f6597 Tenchu goes ingame! As you can see, the graphics is almost flawless, but there is a sort of "beep sound" that repeats itself loudly that makes the game not very enjoyable. To reach the main menu I also have to skip the intros (which don't render BTW, the screen is black when playing them). One oddity: when playing the game, "stop emulation" does absolutely nothing and "start emulation" restarts the game.

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1327338/KrnlDebug.txt)


Main menu
![tenchu](https://user-images.githubusercontent.com/30000751/30783020-f45037ee-a13c-11e7-8541-bd237f571498.PNG)

Option menu
![tenchu options](https://user-images.githubusercontent.com/30000751/30783021-fb3a4e0a-a13c-11e7-8f66-422605b41a6b.PNG)

Select character menu
![tenchu select](https://user-images.githubusercontent.com/30000751/30783022-01097374-a13d-11e7-8e90-4ae812649180.PNG)

Ingame
![tenchu ingame](https://user-images.githubusercontent.com/30000751/30783023-07a28a22-a13d-11e7-8373-d93e1b8b51ee.PNG)

**Comment by ergo720** [comment]
Hmm, I have looked at my dump of the game and realized that it contains 5 xbe's: default, dashupdate, update, France and UK.xbe. According to https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/issues/730 cxbx now can't stop multi-xbe correctly so that should be why I am experiencing the same issue with this title.

**Comment by hotzenplotz3000** [comment]
I've tested it too.
As mentiont before, after launching the default.xbe it switch to uk.xbe or france.xbe depending on language selection of your dashboard.
Game crashes at random locations, so it's not realy playable :D
@ergo720 can you confirm the crashes?
[KrnlDebug_default.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1330417/KrnlDebug_default.txt)

**Comment by ergo720** [comment]
Yes, I have random crashes as well, so the correct label should probably be "partially-working" instead

**Comment by Sn34k** [comment]
I'm still not able to get past crashing, prior to the c609654 update. Could my default xbe. dump be corrupt?

**Comment by LukeUsher** [comment]
The attached debug logs seem to show that there are some unpatched Direct3D functions, stability could be improved with OOVPA improvements.

**Comment by ObiKKa** [comment]
Hey. A [video](https://www.youtube.com/watch?v=9TfUkfWhStw) uploaded 3 days ago by Literalmente{Game}. Whoa, what's so cool about this demonstration is the almost clean graphics and nearly stable gameplay for a brief time. In fact, it's looking closer to the active Dreamcast project _redream_! Luke and Patrick are still on the right pathways.

Much excitement awaits all of us!

**Comment by Sn34k** [comment]
Any updates on progress?

**Comment by hotzenplotz3000** [comment]
Works with current build.
Seems like the random crashes also gone now. Played it for 10 min. But sound is still broken
@Sn34k you have to skip the intro, otherwise it hangs 
[KrnlDebug_6f5016a8.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1447701/KrnlDebug_6f5016a8.txt)

**Comment by Sn34k** [comment]
I dunno what the issue is but i've still yet to get it to load at all. Not sure if its my build or what? It used to load the loading screen and crash now i don't even get that. Just a bunch of errors Direct sound errors. Also are you guys running in compatiblity mode? And lastly where is the lastest build link? I'm curious if my issue is using the automated builds? Let me know, Thanks!

**Comment by literalmente-game** [comment]
After https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/commit/5ec496287df1454f39c0eca6ffd9e63638d52be0 the game isn't able to even get to the menus anymore.

**Comment by Sn34k** [comment]
Any updates?

**Comment by Abacabb69** [comment]
Hey guys, do we have any updates on this issue by chance? Especially since the xbox leaks n all?

**Comment by Abacabb69** [comment]
Latest build and after getting to the main menu and pressing to start story mode, it crashes on this:

![Tenchucrash01](https://user-images.githubusercontent.com/66146247/83942514-eb984700-a7eb-11ea-8203-a8dc7e79937b.jpg)
